src/lib/config.ts

```diff
@@ -66,6 +66,12 @@ export const CONFIG: Config = {
             snapshotTests: { present: testsData.hasSnapshotTests, value: 2 },
           }),
         },
+        {
+          name: 'authorTrackRecord',
+          weight: 3,
+          description: 'Track record of strong authors',
+          benchmarks: (packageCount: number) => categorizeHigherIsBetter([20, 11, 5, 2], packageCount),
```

Comment:
track record doesn't make sense to me. the number you are returning is package count, not track record. i can see how it would correlate, but calling the number track record is confusing.